bulbs/poll/models.py

```diff
@@ -0,0 +1,220 @@
+from bulbs.content.filters import Published
+from bulbs.content.models import Content, ContentManager, ElasticsearchImageField
+from bulbs.utils import vault
+
+from django.db import models, transaction
+from django.utils import timezone
+
+from djbetty import ImageField
+
+from djes.models import Indexable
+from elasticsearch_dsl.filter import Range
+from rest_framework.exceptions import APIException
+
+from time import mktime
+import pytz
+import requests
+
+SODAHEAD_DATE_FORMAT = '%m/%d/%y %I:%M %p'
+
+SODAHEAD_POLL_ENDPOINT = 'https://onion.sodahead.com/api/polls/{}/'
```

Comment:
base URL should be defined in settings (ex: settings.`SODAHEAD_BASE_URL = "https://onion.sodahead.com"`)